prog/prog.go

```diff
@@ -207,8 +207,8 @@ func (arg *GroupArg) Size() uint64 {
 		for _, fld := range arg.Inner {
 			size += fld.Size()
 		}
-		if typ.AlignAttr != 0 && size%typ.AlignAttr != 0 {
-			size += typ.AlignAttr - size%typ.AlignAttr
+		if typ.TypeAlign != 0 && size%typ.TypeAlign != 0 {
```

Comment:
Can we ever have TypeAlign == 0? If not, then remove this b/c this will confuse readers and hide bugs.